script.js

```diff
@@ -0,0 +1,95 @@
+let defaultResult = 0;
+
+let currentResult = defaultResult;
+
+let logEntries = [];
+
+function getUserInput() {
+  return parseInt(userInput.value);
+}
+
+// the description & result output of calculation
+
+function createOutput(numberBeforeCalc, operator, calcNumber) {
+  const calcDescription = `${numberBeforeCalc} ${operator} ${calcNumber}`;
+  outputResult(currentResult, calcDescription);
+}
+
+// object inside array
+
+function writeToLog(
+  operationIdentifier,
+  prevResult,
+  numberEnteredByUser,
+  finalResult
+) {
+  const logEntry = {
+    operation: operationIdentifier,
+    PrevResult: prevResult,
+    number: numberEnteredByUser,
+    output: finalResult,
+  };
+
+  logEntries.push(logEntry);
+  console.log(logEntries);
+}
+
+// using if statement instead of seperate functions
+
+function calculationResult(calculationType) {
```

Comment:
function should be treated as verb
can we name it as `calculateResult` ?

Reply:
Done @faran4engg